### ISSUE TITLE
Cop idea: Move a `return` from an one-line multi-block iterator

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12107]
## Actual behavior

I have a `return` into an one-line multi-block iterator:

```ruby
# frozen_string_literal: true

module ApplicationHelper
  def my_method(fields)
    fields.each do |field|
      return 'active' if condition?(field)
    end
  end
end
```

## Expected behavior:

```ruby
# frozen_string_literal: true

module ApplicationHelper
  def my_method(fields)
    return 'active' if fields.any? { |field| condition?(field) }
  end
end
```

## RuboCop version

```bash
ydakuka@yauhenid:~/Work/project$ bin/rails_docker rubocop -V
1.55.1 (using Parser 3.2.2.3, rubocop-ast 1.29.0, running on ruby 2.7.8) [x86_64-linux]
  - rubocop-capybara 2.18.0
  - rubocop-factory_bot 2.23.1
  - rubocop-performance 1.18.0
  - rubocop-rails 2.20.2
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.23.0
  - rubocop-thread_safety 0.5.1
```

**Comment by larouxn** [comment]
Quick thought, we shouldn't need the `return` since it'll implicitly return `active` or `nil`.

```ruby
# frozen_string_literal: true

module ApplicationHelper
  def my_method(fields)
    'active' if fields.any? { |field| condition?(field) }
  end
end
```
